bentoml/adapters/multi_image_input.py

```diff
@@ -36,64 +47,133 @@ class MultiImageInput(BaseInputAdapter):
             https://imageio.readthedocs.io/en/stable/format_png-pil.html
 
     Raises:
-        ImportError: imageio package is required to use ImageInput
+        ImportError: imageio package is required to use MultiImageInput
 
     Example usage:
-        ```
-        class MyService(BentoService):
-            @bentoml.api(input=MultiImageInput(input_names=('imageX', 'imageY')))
-            def predict(self, image_groups):
-                for image_group in image_groups:
-                    image_array_x = image_group['imageX']
-                    image_array_y = image_group['imageY']
-        ```
+
+    >>> from bentoml import BentoService
+    >>> import bentoml
+    >>>
+    >>> class MyService(BentoService):
+    >>>     @bentoml.api(input=MultiImageInput(input_names=('imageX', 'imageY')))
+    >>>     def predict(self, image_groups):
+    >>>         for image_group in image_groups:
+    >>>             image_array_x = image_group['imageX']
+    >>>             image_array_y = image_group['imageY']
+
+    The endpoint could then be used with an HTML form that sends multipart data, like
+    the example below
+
+
+    >>> <form action="http://localhost:8000" method="POST"
+     enctype="multipart/form-data">
```

Comment:
Looks like this line will break the formatting? this docstring will be used to generate the API docs page, would be nice if you could build the docs locally and check the docs formatting, here are related docs for doing that: https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#how-to-edit-run-build-documentation-site